projects/igniteui-angular/src/lib/progressbar/progressbar.component.ts

```diff
@@ -402,6 +401,18 @@ export class IgxLinearProgressBarComponent extends BaseProgressDirective impleme
     constructor() {
         super();
     }
+    public set valueInPercent(value: number) {
+        const val = toValue(value, this._max);
+        if (value < 100) {
+            this.value = val;
+        } else {
+            this.value = 100;
+        }
+    }
+
+    public get valueInPercent(): number {
```

Comment:
`get valueInPercent` seems the same for both extended classes. Maybe just move it under the base class.